### ISSUE TITLE
About hdf5 files

### ISSUE THREAD

**Issue by Patrickcxt** [syyeung/frameglimpses#1]
Hi， can you please provide some scripts to generate the hdf5 files such as train_data_file? thx~

**Comment by buvaev** [comment]
@Patrickcxt , this might be helpful 
https://github.com/duguyue100/neuralfeature
https://github.com/garythung/torch-lrcn

**Comment by wandering007** [comment]
@Patrickcxt @buvaev Hi, does anyone know where to get the temporal annotations of training data?

**Comment by cloudandcat** [comment]
Does anyone run successfully following this project? I have a little data format question.

**Comment by bityangke** [comment]
@cloudandcat I also have some question, for example, are all the clips are used for training?

**Comment by cloudandcat** [comment]
@bityangke I have prepared for data files，but when i trained it ， i got some problems.

**Comment by bityangke** [comment]
@cloudandcat What is the problem？ Cannot get good results or what?

**Comment by cloudandcat** [comment]
@bityangke I think that is the problem of my cutorch. I couldnt run the code successfully. If you want to check I can send the data file to you.

**Comment by bityangke** [comment]
@cloudandcat  Could you put the sample of your training meta file here?

**Comment by wandering007** [comment]
@cloudandcat I found that most  of UCF101 videos are less than 10 seconds. If they are sampled at 5fps, we cannot get 50 frames...

**Comment by bityangke** [comment]
@wandering007 Indeed， I solve this by padding frames

**Comment by cloudandcat** [comment]
@wandering007  The official website of Thumos2014 has the dataset which is 78GB including the temporal labeling

**Comment by wandering007** [comment]
@bityangke Padding what frames? last one? I thought about it, but finally gave up. I think that it would bring incorrect temporal info.

**Comment by cloudandcat** [comment]
@wandering007 方便邮件沟通吗？I think there is the problem of my cutorch. I have prepared for
the data.

**Comment by cloudandcat** [comment]
@bityangke  I prepared the meta file following with the Readme file

**Comment by bityangke** [comment]
@buvaev What step do you choose？ No overlapping？
And all the clips are used？ How about the Non-action clips？

**Comment by bityangke** [comment]
@cloudandcat We can exchange idea about action recognition, My QQ: 1120339906, plz send me a message.

**Comment by wangxiao5791509** [comment]
@bityangke Hi, would you please tell me how to pre-process the UCF101 dataset ? i.e. how to transform the *.avi video to hdf5 files ??? Thank you.

**Comment by Gandhi-Sagar** [comment]
Does anyone know how to generate meta file with THUMOS-14? The training data contains the trimmed videos. So the ground truth will always be 0 and last_frame_no, right? Also, do we need to use background data? Could we collectively work towards closing the issue? May be convey what we know? I think I have figured out how to generate hdf5 files. Will update based on a success..

**Comment by June01** [comment]
@Gandhi-Sagar Could you share your idea about how to generate hdf5 files? If needed, I can also help to work with meta data or others that haven't finished. 
@syyeung Could you please give some reference ahout how to generate the data as you write? It will be a great help in this problem if you provide them all. Thank you very much~
